packages/discord.js/src/structures/interfaces/TextBasedChannel.js

```diff
@@ -78,6 +78,7 @@ class TextBasedChannel {
    * @typedef {BaseMessageOptions} BaseMessageCreateOptions
    * @property {boolean} [tts=false] Whether the message should be spoken aloud
    * @property {string} [nonce=''] The nonce for the message
+   * @property {boolean} [enforceNonce] Whether the message should check for uniqueness
```

Comment:
I don't think this description is very descriptive of what is actually happening, but I don't have better suggestions. This should mention, at least, that if another message with the same nonce was posted, that message will be returned instead of the new one.